mula/.ci/docker-compose.yml

```diff
@@ -24,20 +24,23 @@ services:
     env_file:
       - .ci/.env.test
     depends_on:
-      ci_postgres:
-        condition: service_healthy
-      ci_rabbitmq:
-        condition: service_healthy
+      - ci_postgres
+      - ci_bytes
+      - ci_katalogus
+      - ci_rabbitmq
 
   ci_postgres:
     image: postgres:15
     healthcheck:
-      test: ["CMD", "pg_isready"]
+      test: ["CMD", "gosu", "postgres", "pg_isready"]
       interval: 10s
-      timeout: 5s
-      retries: 5
+      retries: 10
     env_file:
       - .ci/.env.test
+    environment:
+      APPS: "BYTES KATALOGUS SCHEDULER"
+    volumes:
+      - ../init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
```

Comment:
If I'm not mistaken it should be one directory up?

Reply:
I think since the make file is being called from the mula folder and as such the docker-compose command from that context it needs to go one folder up in order to add the init-user-db.sh script